src/capture_RPiHQ.cpp

```diff
@@ -847,73 +481,76 @@ if (extraFileAge == 99999 && ImgExtraText[0] == '\0') ImgExtraText = "xxxxxx kee
-		min_saturation    		= 0.0;
-		max_saturation    		= 2.0;
+		saturation				= default_saturation;
+		min_saturation			= 0.0;
+		max_saturation			= 2.0;
 
 		default_brightness		= 0;
-		asiDayBrightness  		= default_brightness;
+		asiDayBrightness		= default_brightness;
 		asiNightBrightness		= default_brightness;
-		min_brightness    		= -100;
-		max_brightness    		= 100;
+		min_brightness			= -100;
+		max_brightness			= 100;
 	}
 	else
 	{
 		default_saturation		= 0.0;
-		saturation        		= default_saturation;
-		min_saturation    		= -100.0;
-		max_saturation    		= 100.0;
+		saturation				= default_saturation;
+		min_saturation			= -100.0;
+		max_saturation			= 100.0;
 
 		default_brightness		= 50;
-		asiDayBrightness  		= default_brightness;
+		asiDayBrightness		= default_brightness;
 		asiNightBrightness		= default_brightness;
-		min_brightness    		= 0;
-		max_brightness    		= 100;
+		min_brightness			= 0;
+		max_brightness			= 100;
 	}
-	int asiRotation       		= 0;
-	char const *latitude  		= "52.57N";
-	char const *longitude 		= "4.70E";
-	char const *angle     		= "0"; // angle of the sun with the horizon (0=sunset, -6=civil twilight, -12=nautical twilight, -18=astronomical twilight)
-	int preview           		= 0;
-	int showTime          		= 0;
-	int showExposure      		= 0;
-	int showGain          		= 0;
-	int showBrightness    		= 0;
-	int showMean          		= 0;
-	int showFocus         		= 0;
-	int darkframe         		= 0;
-	int daytimeCapture    		= 0;
-	int help              		= 0;
-	int quality           		= 90;
+	int asiRotation				= 0;
+	char const *latitude		= DEFAULT_LATITUDE;
+	char const *longitude 		= DEFAULT_LONGITUDE;
+	// angle of the sun with the horizon
+	// (0=sunset, -6=civil twilight, -12=nautical twilight, -18=astronomical twilight)
+	char const *angle			= DEFAULT_ANGLE;
+
+	int preview					= 0;
+	int showTime				= DEFAULT_SHOWTIME;
+	int showExposure			= 0;
+	int showGain				= 0;
+	int showBrightness			= 0;
+	int showMean				= 0;
+	int showFocus				= 0;
+	int darkframe				= 0;
+	int daytimeCapture			= 0;
+	int help					= 0;
+	int quality					= 90;
 
 	int i;
-	bool endOfNight    			= false;
+	bool endOfNight				= false;
 	int retCode;
 	cv::Mat pRgb;	// the image
 
 	//-------------------------------------------------------------------------------------------------------
 	//-------------------------------------------------------------------------------------------------------
-	setlinebuf(stdout);   // Line buffer output so entries appear in the log immediately.
+	setlinebuf(stdout);		// Line buffer output so entries appear in the log immediately.
 	if (setlocale(LC_NUMERIC, locale) == NULL)
 		printf("*** WARNING: Could not set locale to %s ***\n", locale);
 
 	printf("\n");
 	printf("%s ********************************************\n", c(KGRN));
-	printf("%s *** Allsky Camera Software v0.8.3 | 2021 ***\n", c(KGRN));
+	printf("%s *** Allsky Camera Software v0.8.3.2 | 2021 ***\n", c(KGRN));
```

Comment:
with
`CFLAGS += -DVERSION_BUILD=\"v0.8.3.2 | 2022\"
`
in Makefile
and 
printf("*** Allsky Camera Software %s ***\n", VERSION_BUILD);

you have only one place to change

Reply:
@AndreasLMeg I definitely need to update the year.
On Dicord we had talked about going to a date-based version, but I think that should be for a build number, not the whole version.  For example, version 0.8 build 2022-02-20.  This would imply that both capture programs could have different versions.  We would need to hard-code the date or else it would be whatever date the user compiled the code.
When we deploy an apt package we can add the date to the makefile.